src/server/send/sendAPI.js

```diff
@@ -71,15 +72,21 @@ const setup = (app: Router) => {
   app.post(
     '/send/recoveryinstructions',
     passport.authenticate('jwt', { session: false }),
-    onlyInEnv('production', 'staging', 'test'),
+    onlyInEnv('production', 'staging', 'test', 'development'),
     wrapAsync(async (req, res, next) => {
       const log = req.log.child({ from: 'sendAPI - /send/recoveryinstructions' })
       const { user } = req
-      const { mnemonic } = req.body
-
-      log.info('sending recovery email', user)
+      const { magicLine } = req.body
+      let userRec = user
+      if (!user.mauticId || user.mauticId < 0) {
+        const mauticContact = await Mautic.createContact(userRec)
+        userRec.mauticId = mauticContact.contact.fields.all.id
```

Comment:
you need to update user record with his mauticid

Reply:
[done](https://github.com/GoodDollar/GoodServer/pull/69/commits/364657f74a0afdedaf1845aa983d9e7450ff937a)